packages/clay-sticker/src/index.tsx

```diff
@@ -54,8 +54,37 @@ interface IProps extends React.HTMLAttributes<HTMLSpanElement> {
 	 * Sticker size.
 	 */
 	size?: Size;
+
+	/**
+	 * Flag to indicate if `sticker-user-icon` class should be applied.
+	 */
+	userIcon?: boolean;
```

Comment:
This name seems a bit strange when you use it with any Sticker and Image, the icon of a user always comes to my mind, it seems very specific. Looking at the class it looks like it just rounds the edges and some details, maybe the name of the API is better with `circle`, what do you think?